packages/expo-store-review/src/ExpoStoreReview.ts

```diff
@@ -1,13 +1,2 @@
-import { Platform } from '@unimodules/core';
-
-export default {
-  get name(): string {
-    return 'ExpoStoreReview';
-  },
-  async isAvailableAsync(): Promise<boolean> {
-    // true on Android, false on web
-    return Platform.OS === 'android';
-  },
-  // Unimplemented on web and Android
-  requestReview: null as null | (() => Promise<void>),
-};
+import { NativeModulesProxy } from '@unimodules/core';
```

Comment:
Make this file `ExpoStoreReview.native.ts` and convert `ExpoStoreReview.web.ts` back to `ExpoStoreReview.ts` because TypeScript will resolve the types from the `.ts` file.